lib/routes.jsx

```diff
@@ -115,6 +115,8 @@ var routeElements = Object.keys(pages).map(function(path) {
   return <Route path={path} component={pages[path]} key={path}/>;
 });
 
+var makerPartyLocales = ['de', 'fr', 'es', 'pl', 'it', 'nl', 'cs', 'bg'];
```

Comment:
@alicoding How does learning.mozilla.org handle this? If fr and fr-FR are the same (I'm assuming this, but because I've never come across fr-FR I'm not sure) we need to flatten these codes to match whatever is stored in the repo (and pontoon) so does learning.mozilla.org at some point do a locale code convert, or does it store two locale files that are similar for each code? Or, otherwise, how does this work?